### DISCUSSION TITLE
Demo page examples

### DISCUSSION THREAD

**Discussion by JustinZz7** [swup/swup#967] [General]
Examples just say not found 

https://swup.js.org/getting-started/demos/

**Comment by daun** [comment]
[Replit](https://replit.com/), the service we're using to host the demos, has recently disabled free hosting and we don't have a budget for paying them. We're evaluating other options for hosting the demos, but haven't settled on anything yet. Let us know if you have suggestions, we'd love to explore some more options.

**Comment by hirasso** [comment]
Thank you @JustinZz7 for poking us about this! 

I've just ported one of the replits to https://glitch.com. It felt nice and easy and seems to be embeddable as well! 

![CleanShot 2024-10-24 at 09 55 43@2x](https://github.com/user-attachments/assets/2cebf00c-1de5-422a-895c-c0193b9820b1)

Code: https://glitch.com/edit/#!/pricey-spiritual-anorak
Preview: https://pricey-spiritual-anorak.glitch.me/ (we could use this to embed it on the docs 🥳)

I love the overall simplicity and friendly vibe of the tool. The only way to find out if it works would be to port all our demos over... 

@daun I don't have access to the swup team replits anymore, so I'm afraid that would be something only you could do? 🥵

Also, it would be beautiful if we all would be able to use glitch with a shared account. ~So we maybe should create the demos using a login that's email- instead of github-based. What do you think?~ Actually, it looks like one can invite other glitch users to collaborate on a project (See screenshot above, "**Invite project members** to edit your code.")

**Reply by hirasso** [reply]
This is the source code of the [demo page](https://pricey-spiritual-anorak.glitch.me/):

```html

<!DOCTYPE html>
<html>

<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width">
  <title>Swup Demo</title>
  <link rel="stylesheet" href="base.css" />
  <link rel="stylesheet" href="transitions.css" />
  <script type="module" src="swup.js"></script>
</head>

<body>
  <div class="layout">
    <header class="header">
      <a href="https://swup.js.org" target="_blank" class="logo">Swup Demo</a>
    </header>
    <main id="swup" class="transition-main">
      <h1>
        <span>Fade Animations</span>
        <span>With Swup</span>
      </h1>
      <p>
        A wonderful serenity has taken possession of my entire soul, like these
        sweet mornings of spring which I enjoy with my whole heart. I am alone,
        and feel the charm of existence in this spot, which was created for the
        bliss of souls like mine. I am so happy, my dear friend.
      </p>
      <p class="buttons">
        <a href="/next.html" class="button">Next page  →</a>
      </p>
    </main>
  </div>
</body>

</html>
```

No history augmentation as with CodeSandbox, no other cruft. I vote for Glitch!

EDIT
Pasted the source again, without formatting, which lead to the markdown-style attributes

**Comment by hirasso** [comment]
We migrated all our demos from Replit to Glitch. Back online 🤠🤠
